js/index.js

```diff
@@ -93,3 +110,31 @@ messageSection.hidden = true;
 
 /* add event listener to submit button with callback function */
 messageForm.addEventListener("submit", formSubmit);
+
+/* created GET request for github repos */
+fetch("https://api.github.com/users/alcarrillomunoz/repos")
+    .then(response => {
+        if (!response.ok) {
+            throw new Error('Request failed');
+          }
+        return response.json();
+    })
+    .then(data => {
+        /* create respositories element to store array */ 
+        repositories = [...data];
+
+        /* create project section selector  */
+        const projectSection = document.getElementById("projects"); 
+        const projectList = projectSection.querySelector("ul");
+
+        /* loop through repositories array and add list of repositories to projects section */
+        for (let i = 0; i < repositories.length; i++) {
+            var project = document.createElement("li"); 
```

Comment:
Using `var` in JavaScript is generally discouraged in modern development because it has some quirks that can lead to bugs, particularly with how it handles scope and hoisting. Instead, `let` and `const` are preferred.